.stylelintrc.json

```diff
@@ -16,20 +16,25 @@
     "function-name-case": "lower",
     "function-parentheses-newline-inside": "always-multi-line",
     "function-parentheses-space-inside": "never",
-    "function-url-data-uris": "never",
+    "function-url-data-uris": null,
     "function-url-quotes": "always",
     "function-whitespace-after": "always",
-    "number-leading-zero": "never",
+    "number-leading-zero": "always",
     "number-max-precision": 10,
+    "number-no-trailing-zeros": true,
     "string-no-newline": true,
     "string-quotes": "single",
     "length-zero-no-unit": true,
-    "time-no-imperceptible": true,
+    "time-no-imperceptible": [true, {
```

Comment:
Probably will never need a time faster than 100ms but just in case I made this a warning to call attention to when someone does go under 100ms. It's a way of saying, "Are you sure you want this timing?"